pipeline/conftest.py

```diff
@@ -102,3 +102,16 @@ def mock_scraped_df():
         "url" : " realurl.com",
         "body" : "body2"
         }])
+
+
+@pytest.fixture
+def mock_loading_df():
+    """Fixture for mock df to be used in testing load"""
+    return pd.DataFrame([{
+        "title" : "title",
+        "url" : "test.com",
```

Comment:
Line 112 using this fixture would fail as there is a constraint on the url that it contains "www."